ostia-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: apis.ostia.3scale.com
```

Comment:
3scale.net 😝

Reply:
Naming.. so difficult.

I'm even not sure about the whole "ostia-operator"... "api-operator"?

Reply:
I guess we will have just one operator, so ostia-operator is fine for me.